e2e/cypress/integration/onboarding/tutorial_guided_tour_spec.js

```diff
@@ -9,66 +9,48 @@
 
 // Stage: @prod
 // Group: @onboarding @smoke
+const selectors = {
+    tip: '#tipButton',
+    tipNext: '#tipNextButton',
+    tipText: '.tip-overlay',
+}
 
 describe('Onboarding', () => {
-    let testUser;
-    let otherUser;
-    let testTeam;
-
-    // let config;
-
     before(() => {
-        // cy.apiGetConfig().then((_data) => {
-        cy.apiGetConfig().then(() => {
-            // ({config} = data);
-        });
-
-        // # Set Support Email setting
-        // const newSettings = {
-        //     SupportSettings: {
-        //         SupportEmail: 'feedback@mattermost.com',
-        //     },
-        // };
-        // cy.apiUpdateConfig(newSettings);
-
         // # Create new team and new user and visit Town Square channel
         cy.apiInitSetup().then(({team}) => {
-            testTeam = team;
+            cy.apiCreateUser({bypassTutorial: false}).then(({user}) => {
+                cy.apiAddUserToTeam(team.id, user.id);
 
-            cy.apiCreateUser({bypassTutorial: false}).then(({user: user2}) => {
-                otherUser = user2;
-                cy.apiAddUserToTeam(testTeam.id, otherUser.id);
-            });
-
-            cy.apiCreateUser({bypassTutorial: false}).then(({user: user1}) => {
-                testUser = user1;
-                cy.apiAddUserToTeam(testTeam.id, testUser.id);
-
-                cy.apiLogin(testUser);
-                cy.visit(`/${testTeam.name}/channels/town-square`);
+                cy.apiLogin(user);
+                cy.visit(`/${team.name}/channels/town-square`);
             });
         });
     });
 
     it('Takes the user through the steps of using the app', () => {
         // tip 1: post message
-        cy.get('#create_post #tipButton').click();
-
-        cy.get('.tip-overlay').should('be.visible').
-            and('contain', 'Type your first message and select Enter to send it.');
-
-        cy.get('#tipNextButton').click();
+        cy.get(`#create_post ${selectors.tip}`).click();
```

Comment:
We can add comments on every code block to tell what the intended action is like the existing e2e files